### ISSUE TITLE
[BUG] Named export 'Appender' not found.

### ISSUE THREAD

**Issue by VictoriqueMoe** [TypedProject/ts-express-decorators#2620]
### Describe the bug

After updating `tsed` to version `7.63.1` when starting it, i get the following error: 
```
SyntaxError: Named export 'Appender' not found. The requested module '@tsed/logger' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:
```
I am using ESM, the error it complains is from `/node_modules/@tsed/logger-file/lib/esm/FileAppender.js` but it seems `FileAppender` has not changed in 2 years, so i am not sure the origin of this issue. i tried checking the changelogs and can't see anything that would cause this.

### To Reproduce

* use logger
* try to start tsed

### Expected behavior

it works

### OS

WIndows 11

### Node version

v20.11.0

### Library version

7.63.1

### Additional context

I was updating from `7.62.3` to `7.63.1`

**Comment by Romakita** [comment]
Hello @VictoriqueMoe 
I recently release a change on the package.json to export precompiled file for browser env. it seems to be related. Probably the exported file for esm is wrong. I’ll check that.

**Comment by Romakita** [comment]
@VictoriqueMoe  fixed ;)
Release v6.7.4

See you !

**Comment by VictoriqueMoe** [comment]
> @VictoriqueMoe fixed ;) Release v6.7.4
> 
> See you !

Thank you as always!

**Comment by VictoriqueMoe** [comment]
I can confirm that after updating, it works well :)
